lib/util/emitDeprecation.js

```diff
@@ -0,0 +1,9 @@
+const emittedCodes = new Set();
+
+module.exports = function emitDeprecation(code, description) {
+    if(emittedCodes.has(code) ) {
+        return;
+    }
+    emittedCodes.add(code);
+    process.emitWarning(description, "DeprecationWarning", code);
+};
```

Comment:
Hopefully this matches what you meant by [this](https://github.com/abalabahaha/eris/discussions/1274#discussioncomment-1285642) @bsian03?

I've opted to use a `Set` because a quick search seemed to indicate `Set#has` is more performant than `Array#includes`, though the number of warnings we'll have is probably never going to be big enough for it to truly matter. Could use an array instead for simplicity.

Reply:
See my discussion comment

Reply:
Performance comparison done and discussed on Discord, https://discord.com/channels/831967755447828491/884930349119991849/885222846815678495, arrays are faster. Addressed in e3313be.